docs/application/native/guides/connectivity/softap.md

```diff
@@ -0,0 +1,341 @@
+# SoftAP
+
+SoftAP means Software Access Point for local area connections without internet access.
```

Comment:
This sentence is fine, but I have a question. Should it be abbreviated when you are explaining.
Or
Can we change as:
SoftAP means software access point for local area connections without internet access.
Please share your opinion.